### PR TITLE
Remove tools.EtcdClient reference from master_test

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: pkg/master/master_test.go [hunk 1]
```diff
@@ -596,23 +597,20 @@ func encodeToThirdParty(name string, obj interface{}) ([]byte, error) {
...
-	return err
+	return s.Set(context.TODO(), etcdtest.AddPrefix(path), data, nil, 0)
 }
 
-// TODO: Convert to storage.Interface.
-func setupEtcdList(client tools.EtcdClient, path string, list []Foo) error {
+func setupEtcdList(s storage.Interface, path string, list []Foo) error {
 	for _, obj := range list {
-		if err := storeToEtcd(client, path+"/"+obj.Name, obj.Name, obj); err != nil {
+		if err := storeToEtcd(s, path+"/"+obj.Name, obj.Name, obj); err != nil {
```
Comment (timothysc) [inline pkg/master/master_test.go]: might want to rename function, given the context.

File: pkg/master/master_test.go [hunk 2]
```diff
@@ -900,7 +895,7 @@ func testInstallThirdPartyResourceRemove(t *testing.T, version string) {
 		etcdtest.AddPrefix("/ThirdPartyResourceData/company.com/foos/default/bar"),
 	}
 	for _, key := range expectedDeletedKeys {
-		_, err := client.Get(key, false, false)
+		_, err := etcdserver.Client.Get(key, false, false)
```
Comment (timothysc) [inline pkg/master/master_test.go]: can't you switch this to storage interface as well here?  Just 1 less thing to switch on client shift.

File: pkg/master/master_test.go [hunk 3]
```diff
@@ -813,7 +809,7 @@ func testInstallThirdPartyAPIDeleteVersion(t *testing.T, version string) {
 	assert.Equal(http.StatusNotFound, resp.StatusCode)
 
 	expectedDeletedKey := etcdtest.AddPrefix("ThirdPartyResourceData/company.com/foos/default/test")
-	_, err = client.Get(expectedDeletedKey, false, false)
+	_, err = etcdserver.Client.Get(expectedDeletedKey, false, false)
```
Comment (timothysc) [inline pkg/master/master_test.go]: can't you switch this to storage interface as well here?  You're not actually parsing the objects only checking existence.

File: pkg/master/master_test.go [hunk 4]
```diff
@@ -596,23 +597,20 @@ func encodeToThirdParty(name string, obj interface{}) ([]byte, error) {
 		ObjectMeta: api.ObjectMeta{Name: name},
 		Data:       serial,
 	}
-	return testapi.Extensions.Codec().Encode(&thirdPartyData)
+	return &thirdPartyData, nil
 }
 
-// TODO: Convert to storage.Interface.
-func storeToEtcd(client tools.EtcdClient, path, name string, obj interface{}) error {
+func storeToEtcd(s storage.Interface, path, name string, obj interface{}) error {
```
Comment (timothysc) [inline pkg/master/master_test.go]: might want to rename function "storeToEtcd", given the context.

File: pkg/master/master_test.go [hunk 5]
```diff
@@ -596,23 +597,20 @@ func encodeToThirdParty(name string, obj interface{}) ([]byte, error) {
...
 		return err
 	}
-	_, err = client.Set(etcdtest.PathPrefix()+path, string(data), 0)
-	return err
+	return s.Set(context.TODO(), etcdtest.AddPrefix(path), data, nil, 0)
 }
 
-// TODO: Convert to storage.Interface.
-func setupEtcdList(client tools.EtcdClient, path string, list []Foo) error {
+func setupEtcdList(s storage.Interface, path string, list []Foo) error {
```
Comment (timothysc) [inline pkg/master/master_test.go]: Same comment about removing "Etcd"

**Comment by timothysc** [comment]
minor comments.

File: pkg/master/master_test.go (hunk 2 above)
Comment (wojtek-t) [inline pkg/master/master_test.go]: Done.

File: pkg/master/master_test.go (hunk 5 above)
Comment (wojtek-t) [inline pkg/master/master_test.go]: Done.

File: pkg/master/master_test.go (hunk 4 above)
Comment (wojtek-t) [inline pkg/master/master_test.go]: Done.

**Comment by wojtek-t** [comment]
@timothysc comment applied

PTAL

**Comment by timothysc** [comment]
awesome - lgtm! 

Say, when can I get cool kid credz to apply myself?

**Comment by wojtek-t** [comment]
Self-applying your label.

@lavalamp - can you please add @timothysc to owners?

**Pull request merged**